resourcemonitor/resourcemonitor.go

```diff
@@ -602,6 +615,72 @@ func (monitor *ResourceMonitor) processAlerts() {
 	}
 }
 
+func (xen *xenSystemUsage) RunCmd() {
+	lines := make(chan xentop.Line, 1)
+	errs := make(chan error, 1)
+
+	ctx, cancelFunc := context.WithCancel(context.Background())
```

Comment:
You don't need to create own context with own cancel function, just pass the common context here.

Reply:
Done. I did not know that context could be used in a different goroutine